packages/ui/src/domainModules/curatedList/components/TypeSpecimenTypeDropdownSearch/index.js

```diff
@@ -40,13 +40,15 @@ class TypeSpecimenTypeDropdownSearch extends Component {
       return null
     }
     return (
-      <Grid.Column width={16}>
-        <DropdownSearch
-          {...rest}
-          options={typeSpecimenTypeOptions}
-          type="dropdown-search-local"
-        />
-      </Grid.Column>
+      <Grid>
```

Comment:
I think the fix should be to remove both `Grid` and `Grid.Column` from this dropdown and from `CauseOfDeathDropdownSearch`, because when they are used in the form they are wrapped in a column in `packages/ui/src/domainModules/curatedList/components/formParts/index.js` (and each form part is wrapped in a grid higher up)

can you try that?